entries/t/trovi.sk.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Trovi": {
+    "domain": "trovi.sk",
+    "tfa": [
+      "sms",
+      "email",
+      "totp"
+    ],
+    "documentation": "https://help.trovi.sk/",
```

Comment:
Could you link to the specific page documenting how to enable MFA on the service?